src/renderer/components/dialogs/RenderMedia.js

```diff
@@ -1,17 +1,44 @@
 const React = require('react')
 const styled = require('styled-components').default
+const createGlobalStyle = require('styled-components').createGlobalStyle
```

Comment:
why global style if it's only used here?

Reply:
Because i can't overwrite the bp3-overlay thing without this. I tried to add a wrapper around the <Overlay> component but it didn't show up in the dom. I guess blueprint is doing some stuff there to only use the overlay component.